spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/GrpcServiceRegistrationBean.java

```diff
@@ -33,11 +33,11 @@
  * > public GrpcServiceRegistrationBean helloService() {
  * >     return new GrpcServiceRegistrationBean()
  * >             .setServiceName("helloService")
- * >             .setService(new GrpcServiceBuilder()
- * >                                 .addService(new HelloService())
- * >                                 .supportedSerializationFormats(GrpcSerializationFormats.values())
- * >                                 .enableUnframedRequests(true)
- * >                                 .build())
+ * >             .setService(GrpcService.builder()
+ *    >                                 .addService(new HelloService())
```

Comment:
Spaces were inserted in a wrong place. There must be only one space between `*` and `>`.